src/app/cluster/cluster-details/edit-cluster/component.ts

```diff
@@ -102,6 +103,9 @@ export class EditClusterComponent implements OnInit, OnDestroy {
       [Controls.OPAIntegration]: new FormControl(
         !!this.cluster.spec.opaIntegration && this.cluster.spec.opaIntegration.enabled
       ),
+      [Controls.Konnectivity]: new FormControl(
+        !!this.cluster.spec.clusterNetwork && this.cluster.spec.clusterNetwork.konnectivityEnabled
```

Comment:
If this should be a editable via the edit cluster dialog there are some things missing:
1. add it to the template.html
2. take a look into editCluster() and add it there
3. extend the `ClusterPatch` spec in `shared/entity/cluster.ts`

Otherwise it won't be saved when you hit the `edit` button

Reply:
I added it there but maybe it should be in "Network Configuration" group. I tried to do that too but looked broken. :grimacing:   


![Screenshot from 2021-09-07 17-57-12](https://user-images.githubusercontent.com/1302569/132375612-656dcb56-e432-436b-bf87-1f7106b9719d.png)